src/app.js

```diff
@@ -34,11 +38,29 @@ let tempPath;
  * @param {string} clientId - name that the dump file will actually have on disk.
  * @param {string} uniqueClientId - name that the dump will have on the store.
  */
-async function storeDump(clientId, uniqueClientId) {
+async function storeDump(sinkMeta, uniqueClientId) {
+
+    const {
+        clientId,
+        isJaaSTenant
+    } = sinkMeta;
+
     const dumpPath = `${tempPath}/${clientId}`;
+    const { webhooks: { sendRtcstatsUploaded } = { sendRtcstatsUploaded: false } } = config;
 
     try {
+
+        logger.info('[S3] Storing dump:', uniqueClientId);
```

Comment:
Should we include the `dumpPath` here as well?

Reply:
makes sense, added.